connection-cache/src/nonblocking/client_connection.rs

```diff
@@ -6,7 +6,7 @@ use {
 };
 
 #[async_trait]
-pub trait ClientConnection {
+pub trait ClientConnection: Send + Sync {
```

Comment:
These are the only other required bounds in the end! Other than in the clients, of course